src/components/CarCard/CarCard.tsx

```diff
@@ -1,26 +1,40 @@
 import React, { useState } from "react";
 import "./CarCard.scss";
 import { Card } from "react-bootstrap";
-import { ArrowRight } from "react-bootstrap-icons";
+import Arrow from "../../assets/Arrow.svg";
 import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 
 function CarCard(car: any) {
   const [carDetails] = useState(JSON.parse(JSON.stringify(car)).car);
   const [id] = useState(carDetails.id);
-  const detailsPageLink = "/cardetails/" + id;
+  const detailsPageLink = "/car_details/" + id;
+
+  const navigate = useNavigate();
+  const redirect = () => {
+    navigate(detailsPageLink);
+  };
+
   return (
-    <Card className="cardContainer">
-      <Card.Img variant="top" src={carDetails.image} height={200} />
-      <Card.Body>
-        <Card.Title>{carDetails.name}</Card.Title>
-        <Card.Text>
-          {carDetails.price}akh onwards
-          <Link to={detailsPageLink}>
-            <ArrowRight />
-          </Link>
-        </Card.Text>
-      </Card.Body>
-    </Card>
+    <div className="transition" onClick={redirect} data-testid="card">
+      <Card className="cardContainer ">
+        <Card.Img
+          variant="top"
+          data-testid="carimg"
+          src={carDetails.image}
+          height={200}
+        />
+        <Card.Body>
+          <Card.Title data-testid="cardetail">{carDetails.name}</Card.Title>
+          <Card.Text className="font-styles " data-testid="cardprice">
```

Comment:
what kind of class name is font-styles? 😆 
Please name it appropriately from one of these if possible
This is available in figma file, you can go there by choosing `Components` in the nav bar menu on the left
![Screenshot 2022-04-29 at 11 51 35 AM](https://user-images.githubusercontent.com/104024981/165894081-a8ea9ad2-6a17-46ca-80a5-e770efda8ddd.png)

Reply:
card design has been modified including font size and color along with shadow.
Added box-shadow on hover only.
<img width="345" alt="image" src="https://user-images.githubusercontent.com/97250294/165916574-2bde5cb1-58f9-4fc4-8a31-df86b75fc38a.png">

on hover:
<img width="380" alt="image" src="https://user-images.githubusercontent.com/97250294/165916644-a9c302da-8e62-472d-95a7-f53a2d5568fa.png">

Reply:
why does the font weight of sub text change on hover?